assets/models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -4,17 +4,28 @@ path: ./
 properties:
   SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
   datasets: databricks/databricks-dolly-15k
-  evaluation-min-sku-spec: 16|0|64|128
-  evaluation-recommended-sku: Standard_NC24s_v3
+  evaluation-min-sku-spec: 8|0|28|56
+  evaluation-recommended-sku: Standard_DS4_v2
```

Comment:
@KonakanchiSwathi Could you please share the job url for evaluation that succeeded on DS4? 

Cc: @ShipraJain01